products/coder.md

```diff
@@ -0,0 +1,80 @@
+title: Coder
+category: server-app
+tags: remote cde terraform ide development docker kubernetes
```

Comment:
```suggestion
tags: build-tool
```

Only a limited set of tags is accepted, see https://endoflife.date/recommendations. The closest I could find is `build-tool` for now (we don't have an `ide` nor a `developer-tool` tag). WDYT ?

Reply:
I think. It doesn't come under a `build-tool` what do you think about supporting a new tag like `ide` or/and `development`?

Reply:
Not for now, this should be done in a separate PR if there is enough candidate (we are not adding new tags unless it is the vendor name or applicable to at least three products (see https://endoflife.date/contribute, previously shared the wrong link).

So for now I will just delete the tags (`app` is a category and cannot be used as a tag).

Reply:
I see app being used as a tag in many apps. 
`server-app` is a category.